spec/rack_attack_throttle_spec.rb

```diff
@@ -3,8 +3,9 @@
 require_relative 'spec_helper'
 
 describe 'Rack::Attack.throttle' do
+  freeze_time!
```

Comment:
I tried these changes but it didn't work
<img width="1280" alt="image" src="https://github.com/rack/rack-attack/assets/6373536/fd8f247d-920d-4c5b-8f04-0ee0fe60529b">

I think we need to make sure that `Timecop` gets called right before the request is sent (in another before block).

Reply:
also if it's possible to use `Timecop.freeze do` (with block) [I think I'd prefer it](https://github.com/travisjeffery/timecop#timecopsafe_mode) even if it means that we no longer have the requests performed in a before block.

Reply:
Oh, bah. Minitest::Spec doesn't support defining multiple `before` blocks at the same level 😞

Reply:
Thanks for catching that, I didn't test the timecop version well enough. See what you think to the new version.